test/unit/Support.cpp

```diff
@@ -144,6 +144,12 @@ uint32_t diff_text_files(std::istream& str1, std::istream& str2, int32_t ignoreL
             continue;
         }
 
+        // remove line endings from test comparisons
```

Comment:
Wonder what's going on here and why things are different. The only reason I would care is that line endings are one of these gotcha things that come up from time to time.

Reply:
It is the use of fstreams versus streambuf. If you follow the PDAL LOG test it creates a file in binary mode and writes with lines such as "DEBUG\n" which are being converted by the fstream even in binary mode. The git checkout also corrects the line endings between Windows and Linux (it was easy to verify with my WSL2 setup).
Technically the VSI approach is correct, the file opened as binary should not have line endings corrected.